entities/entities.go

```diff
@@ -83,3 +84,12 @@ type FunctionAuth struct {
 	Token string
 	Type  AuthenticationType
 }
+
+type Deployment struct {
```

Comment:
The `list` command also declares a type for this. I think we could make each command use the same type? https://github.com/capeprivacy/cli/blob/main/cmd/cape/cmd/list.go#L31

Reply:
switched list to use this one